app/src/main/java/com/keylesspalace/tusky/components/compose/MediaPreviewAdapter.kt

```diff
@@ -43,12 +44,16 @@ class MediaPreviewAdapter(
         val item = differ.currentList[position]
         val popup = PopupMenu(view.context, view)
         val addCaptionId = 1
-        val removeId = 2
+        val editImageId = 2
+        val removeId = 3
         popup.menu.add(0, addCaptionId, 0, R.string.action_set_caption)
+        if (item.type == ComposeActivity.QueuedMedia.Type.IMAGE)
+            popup.menu.add(0, editImageId, 0, R.string.action_edit_image)
```

Comment:
In a future PR, I would like to also add an "edit media" item that appears on audio and video, which would call out to an external activity (for example to trim an attached video clip).